app/bundles/LeadBundle/Views/List/form.html.php

```diff
@@ -26,7 +26,7 @@
     'timezones' => 'timezone-template',
     'select'    => 'select-template',
     'lists'     => 'leadlist-template',
-    'emails'    => 'lead_email_received-template',
+    'emails'    => 'lead_email_read-template',
```

Comment:
I'm a bit scared about this change but I could not find any other usage of `lead_email_received-template` in the code